incubator/zookeeper/values.yaml

```diff
@@ -137,7 +112,7 @@ command:
 #   mountPath: /opt/zookeeper/secret
 
 persistence:
-  enabled: true
+  enabled: false
```

Comment:
I was curious as to what would happen when upgrading a deployment of this chart that was originally configured with persistence enabled, with my main concern being that the existing persistent volumes would be deleted.

Fortunately, k8s rejected the change with this error `Error: UPGRADE FAILED: StatefulSet.apps "zookeeper" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden`, preventing any data loss.

Generally, what's the motivation for changing this from `true` to `false`?

Reply:
reverted. IMHO the default is wrong but it was not the intent of this PR to change.